lib/server.js

```diff
@@ -1164,7 +1165,8 @@ Server.prototype._setupRequest = function _setupRequest(req, res) {
     // we consider a closed request as flushed from metrics point of view
     function onReqAborted() {
         // Request was aborted, override the status code
-        var err = new errors.RequestCloseError();
+        //var err = new errors.RequestCloseError();
```

Comment:
Why do we want to keep this comment?

Reply:
I will get rid of it, this is not needed.